src/index.html

```diff
@@ -1,8 +1,10 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="has-navbar-fixed-top">
 <head>
   <meta charset="utf-8">
-  <title>Project2Frontend</title>
+  <title>PrettyPenny</title>
+  <link href="https://use.fontawesome.com/releases/v5.0.6/css/all.css" rel="stylesheet">
```

Comment:
maybe we can move this to an npm dependency? (just npm install font-awesome --save)

(not part of mvp and not top priority)

Reply:
okay. I will do